docs/user-guide/cli.md

```diff
@@ -18,6 +18,13 @@ custom plugins.
 
 #### `--environment=NAME / -n NAME`
 
+*Default: Disabled*
```

Comment:
I think you've got this backwards, this appears to add documentation to the --environment= option.